.github/workflows/build.yml

```diff
@@ -105,7 +105,8 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
     env:
       # Set up wheels matrix.  This is CPython 3.6--3.9 for all OS targets.
-      CIBW_BUILD: "cp3[6-9]-*"
+      CIBW_BUILD: "cp3{6,7,8,9}-*"
```

Comment:
We should build wheels for 3.10 now that we made changes to support it.
Also, 3.6 is no longer supported so we can remove it.